nav2_gps_waypoint_follower_demo/models/turtlebot_waffle_gps/model.sdf

```diff
@@ -0,0 +1,498 @@
+<?xml version="1.0" ?>
```

Comment:
Is there a reason this is separated from the URDF?

Reply:
AFAIK this is how Nav2 simulation is set up. There's an [sdf](https://github.com/ros-planning/navigation2/blob/main/nav2_bringup/worlds/waffle.model) file containing gazebo plugins and a [urdf](https://github.com/ros-planning/navigation2/blob/main/nav2_bringup/urdf/turtlebot3_waffle.urdf) for the robot state publisher

Reply:
OK! No issue then.